exercises/practice/robot-simulator/.meta/example.gd

```diff
@@ -0,0 +1,25 @@
+@export var position : Vector2i
+@export var direction : String
+
+
+var allowed_directions = ["north", "east", "south", "west"]
+
+
+func move(instructions: String):
+	for instruction in instructions:
+		match [instruction, direction]:
+			["L", _]:
+				var index = allowed_directions.find(direction) - 1
+				direction = allowed_directions[index]
+			["R", _]:
+				var index = allowed_directions.find(direction)
+				index = (index + 1) % 4
```

Comment:
For future exercises: Godot has a function called `wrapi` (also `wrapf` and `wrap`), which is very useful in cases like this one. But, as you mentioned, the example solution in here just to prove that an exercise can be solved ;) We can keep it like this ;)

Reply:
D’oh.

Reply:
If we add approaches later, that’d be a good thing to mention.